cmd/screenshot/main.go

```diff
@@ -4,13 +4,13 @@ import (
 	"log"
 	"net/http"
 
-	"github.com/pressly/screenshot/internal/screenshotserver"
+	"github.com/pressly/screenshot/internal/server"
 	"github.com/pressly/screenshot/rpc/screenshot"
 )
 
 func main() {
 
-	srv := &screenshotserver.Server{}
+	srv := &server.Server{}
```

Comment:
Reads a bit funny, I guess that's why they suggest 

> Do not name the package something generic like api, client or service; name it after your service. Remember that the package is going to be imported by other projects that will likely import other clients from other services as well.

Reply:
yeah, ideally it would read `screenshot.Server{}`

Reply:
but whatever .. server won't be used by the project users directly